plugins/modules/eos_l2_interfaces.py

```diff
@@ -358,35 +463,66 @@
 - name: Gather interfaces facts from the device
   arista.eos.l2_interfaces:
     state: gathered
-# output:
-#   gathered:
-#      - name: Ethernet1
-#        mode: trunk
-#        trunk:
-#          native_vlan: 10
-#      - name: Ethernet2
-#        mode: access
-#        access:
-#          vlan: 30
 
+# Module Execution Result:
```

Comment:
let's try to  keep it similar to what we have for others
# Task output:
# ------------

reference: https://github.com/ansible-collections/arista.eos/blob/80f1ed0263d7fee9daf0e71250bf5a6afa95085c/plugins/modules/eos_bgp_address_family.py#L398